src/tests/chat/commands/help.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { expect } from 'chai'
+import sinon from 'sinon'
+import 'mocha'
+
+import { Help } from '../../../chat/commands/help'
+import { EventBus, Events } from '../../../events'
+
+import { CommandRegistry } from '../../../chat/commandRegistry'
+
+beforeEach(() => {
+  CommandRegistry.init()
+})
+
+afterEach(() => {
+  EventBus.eventEmitter.removeAllListeners()
+})
+
+describe('Commands: Help', () => {
+
+  it('should send message to chat', () => {
+    const spy = sinon.spy()
+
+    const emitter = EventBus.eventEmitter
+    emitter.on(Events.OnSay, spy)
+
+    Help()
+    
+    expect(spy.lastCall.lastArg['message']).to.equal('I can respond to the following commands: attention, awesum, blog, conduct, discord, fart, ffl, font, github, giving, hardware, help, heroines, hype, instagram, keyboard, livecoders, pobox, stop, so, theme, twitter, youtube, jsdefender, store')
```

Comment:
I hope this test fails. If not somethings wrong with the test.

Reply:
Disregard the comment above. Let's test in a way that doesn't require us to constantly update when new commands come in.